src/commands/crashes/upload-symbols.ts

```diff
@@ -47,18 +48,30 @@ export default class UploadSymbols extends AppCommand {
   @hasArg
   public sourceMapPath: string;
 
+  @help("Path to a breakpad symbols zip collection.")
```

Comment:
Will this be a collection of zip files? Or a collection of other files that get zipped? (Apologies - I don't have the context to know this so wanted to check that this help text would be clear to users :) )

Reply:
done!

Reply:
i've changed the wording to "Path to a zip file containing Breakpad symbols." if that's not clear or you'd like me to use different wording, just let me know!

Reply:
Ah, I think what was confusing me here was the naming `prepareZipFromSymbols` - whereas in this case really we're just sending it a zip file and it's returning the path we send it (which seems a little odd!)

when it's a breakpad - will it always be a .zip? If so - this help text is fine :)

Reply:
Yep! Breakpad symbols will always be zipped, as mentioned in our documentation here: https://docs.microsoft.com/en-us/appcenter/diagnostics/android-ndk